en/docs/learn/jarm-for-oauth2.md

```diff
@@ -0,0 +1,162 @@
+# JWT-Secured Authorization Response Mode (JARM) for OAuth 2.0
+
+The [JWT-Secured Authorization Response Mode for OAuth 2.0 (JARM) specification](https://bitbucket.org/openid/fapi/src/master/oauth-v2-jarm.md) defines new JWT-based modes to encode OAuth2 authorization responses. These new modes allow clients to request authorization response parameters and additional data in JWT format.
+
+!!! info
+    This is only available as a WSO2 Update from WSO2 Identity Server update level **5.10.216** onwards. See the instructions on [updating WSO2 products](https://updates.docs.wso2.com/en/latest/).
+
+## Authorization flow
+
+Given below is a sample authorization request sent to the authorization endpoint of WSO2 Identity Server.
+
+```bash
+https://<IS_HOME>/oauth2/authorize?prompt=login&scope=openid&redirect_uri=https://<CLIENT-HOST>/redirects/redirect1&client_id=<CLIENT-ID>&response_type=<RESPONSE-TYPE>&response_mode=<RESPONSE-MODE>
```

Comment:
```suggestion
https://<IS_HOME>/oauth2/authorize?prompt=login&scope=openid&redirect_uri=https://<CLIENT_HOST>/redirects/redirect1&client_id=<CLIENT_ID>&response_type=<RESPONSE_TYPE>&response_mode=<RESPONSE_MODE>
```

Shall we replace "-" with "_" for consistency? There are a few other occurrences as well.